java/system-test/system-test-cdk/src/main/java/sleeper/systemtest/cdk/SystemTestClusterStack.java

```diff
@@ -67,23 +68,23 @@
 public class SystemTestClusterStack extends NestedStack {
 
     public SystemTestClusterStack(
-            Construct scope, String id, SystemTestStandaloneProperties properties, SystemTestBucketStack bucketStack) {
+            Construct scope, String id, SystemTestStandaloneProperties properties, SystemTestBucketStack bucketStack, AutoStopEcsClusterTasksStack autoStopEcsClusterTasksStack) {
         super(scope, id);
-        create(properties, properties, properties.toInstancePropertiesForCdkUtils(), bucketStack);
+        create(properties, properties, properties.toInstancePropertiesForCdkUtils(), bucketStack, autoStopEcsClusterTasksStack);
         Tags.of(this).add("DeploymentStack", id);
     }
 
     public SystemTestClusterStack(
             Construct scope, String id, SystemTestProperties properties, SystemTestBucketStack bucketStack,
-            CoreStacks coreStacks, IngestStacks ingestStacks, IngestBatcherStack ingestBatcherStack) {
+            CoreStacks coreStacks, IngestStacks ingestStacks, IngestBatcherStack ingestBatcherStack, AutoStopEcsClusterTasksStack autoStopEcsClusterTasksStack) {
```

Comment:
It looks like several of these parameters are unused. Please can we remove the unused ones?